packages/editable-html/src/plugins/list/__test__/index.test.js

```diff
@@ -0,0 +1,152 @@
+import React from "react";
+
+import {configure} from "enzyme";
+import Adapter from "enzyme-adapter-react-15";
+import renderer from 'react-test-renderer';
+import List from './../../list/index';
+import Util from '../../utils';
+
+configure({adapter : new Adapter() });
+
+describe("ListPlugin", () => {
+  let list, ListPlugin, toolbar;
+  beforeEach(() => {
+    list = require('../index');
+    ListPlugin = list.default;
+    toolbar = list.toolbar
+  });
+
+  describe("deserialize", () => {
+    it("should deserialize list item", () => {
+      let el = {
+        tagName : 'li',
+        childNodes : []
+      }
+      const next = jest.fn();
+      const out = list.serialization.deserialize(el, next);
+
+      expect(out).toMatchObject({object : 'block', type : 'list_item', nodes: undefined});
+    });
+
+    it("should deserialize ul", () => {
+      let el = {
+        tagName : 'ul',
+        childNodes : []
+      }
+      const next = jest.fn();
+      const out = list.serialization.deserialize(el, next);
+
+      expect(out).toMatchObject({object : 'block', type : 'ul_list', nodes: undefined});
+    });    
+
+    it("should deserialize ol", () => {
+      let el = {
+        tagName : 'ol',
+        childNodes : []
+      }
+      const next = jest.fn();
+      const out = list.serialization.deserialize(el, next);
+
+      expect(out).toMatchObject({object : 'block', type : 'ol_list', nodes: undefined});
+    });        
+  });
+
+  describe("serialize", () => {
```

Comment:
As above - lot of boilerplate - can we shrink it w/ a shared function?

Reply:
Done